owsi-core/owsi-core-components/owsi-core-component-wicket-more/src/main/java/fr/openwide/core/wicket/more/notification/service/AbstractNotificationContentDescriptorFactory.java

```diff
@@ -38,6 +64,20 @@ public Component get() {
 					},
 					locale
 			);
+			
+			
+			IHtmlNotificationCssRegistry cssRegistry = null;
+			if (cssService.hasRegistry(getVariation())) {
+				try {
+					cssRegistry = cssService.getRegistry(getVariation());
+				} catch (ServiceException e) {
+					LOGGER.error("Unable to load the CSS file", e);
```

Comment:
Yeah well, this might lead to an error visible to the user while the rest of the operation is a success so I thought it might as well simply be logged. The error is also highly unlikely so I didn't think it mattered so much.

IMHO, we should work on having a logstash/kibana instance to be able to notice this sort of error.